### PR TITLE
Pull sql

### PR DESCRIPTION
**Pull request by LeroyGorn**

### CODE REVIEW

File: flask_sql.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+@app.route('/customers/')
+def customers():
+    params = {
+        'id':  ' CustomerID {} {} OR',
+        'fax': ' Fax {} OR',
+        'country': ' Country {} {} OR'
+    }
+    query = 'SELECT FirstName, LastName FROM customers'
+    if request.args:
+        query += ' WHERE'
```
Comment (m-seredenko) [inline flask_sql.py:27]: What will I get when send query param that is not a id, fax or country?

File: flask_sql.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+    for k in request.args.keys():
+        if k == 'fax':
+            value = request.args.get(k).replace('_', ' ').upper().strip()
+            query += params[k].format(value)
+            continue
+        if len(request.args.getlist(k)) == 1:
```
Comment (m-seredenko) [inline flask_sql.py:34]: What will happen when I send the next query ?id=1&id=2?

File: flask_sql.py [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+            sign = '='
+            value = "'{}'".format(request.args.get(k))
+        else:
+            sign = 'IN'
+            value = tuple(request.args.getlist(k))
+        query += params[k].format(sign, value)
+
+    data = cursor.execute(query[:-3])
```
Comment (m-seredenko) [inline flask_sql.py:42]: What will happen if no `request.args` was sent?

File: flask_sql.py [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+        query += ' WHERE'
+
+    for k in request.args.keys():
+        if k == 'fax':
+            value = request.args.get(k).replace('_', ' ').upper().strip()
+            query += params[k].format(value)
+            continue
+        if len(request.args.getlist(k)) == 1:
+            sign = '='
+            value = "'{}'".format(request.args.get(k))
```
Comment (m-seredenko) [inline flask_sql.py:36]: What will happen if id was given by mistake as a word?

File: flask_sql.py (hunk 3 above)
Comment (LeroyGorn) [reply flask_sql.py:39]: It will return all names from customers table.

**Pull request merged**